spec/rubocop/cop/rspec/factory_bot/create_list_spec.rb

```diff
@@ -55,7 +55,13 @@
 
     it 'ignores n.times with argument' do
```

Comment:
Unfortunately, we have no coverage for the case we pass an argument, but it not a method call, do you mind adding a couple of such cases, probably as a single example?
```ruby
3.times { |n| create :user, :active }
3.times { |n| create :user, password: '123' }
3.times { |n| create :user, :active, password: '123' }
```
```ruby
    it "flags n.times when create call doesn't have method calls" do
      expect_offense(<<~RUBY)
        3.times { |n| create :user, :active }
        ^^^^^^^ Prefer create_list.
        3.times { |n| create :user, password: '123' }
        ^^^^^^^ Prefer create_list.
        3.times { |n| create :user, :active, password: '123' }
        ^^^^^^^ Prefer create_list.
      RUBY
    end
```
fails.

Reply:
indeed my bad